File: from-3001-to-3100/3005-count-elements-with-maximum-frequency/solution.go
func maxFrequencyElements(nums []int) int {
    cnt := make(map[int]int)
    maxFreq := 0
    for _, num := range nums {
        cnt[num] += 1
        if cnt[num] > maxFreq {
            maxFreq = cnt[num]
        }
    }

    result := 0
    for _, freq := range cnt {
        if freq == maxFreq {
            result += maxFreq
        }
    }
    return result
}

File: from-3001-to-3100/3005-count-elements-with-maximum-frequency/solution.py
class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        maxFreq = 0
        cnt = {}
        for num in nums:
            cnt[num] = cnt.get(num, 0) + 1
            if cnt[num] > maxFreq:
                maxFreq = cnt[num]

        result = 0
        for _, freq in cnt.items():
            if freq == maxFreq:
                result += freq

        return result